### DISCUSSION TITLE
🙏  /ACK connection

### DISCUSSION THREAD

**Discussion by davidxgames** [darthcloud/BlueRetro#216] [Q&A]
Can BlueRetro in Ps2 mode work without the /ACK connection?, I made a Ps2 to xbox control adapter project, but it doesn't use the /ACK connection and therefore doesn't work with BlueRetro, I know this would add latency but I want to try it.

🙏 Could  you please, if it is possible that BlueRetro in Ps2 mode does not use /ACK?

Thanks for reading me.

Note: I use google translator, sorry for the inconvenience.

**Comment by darthcloud** [comment]
ACK is an output from BlueRetro, and so BlueRetro doesn't care if it's used or not.

So if it doesn't work its for another reason.

**Reply by davidxgames** [reply]
BlueRetro I tried it on a ps2 and it works, I put the ps2 controller to ogxbox adapter together and tried it with a ps2 controller and it works.

I tried BlueRetro and the ps2/ogxbox adapter together and it didn't work, the ps5 control doesn't pair.

I will do more tests, thank you very much for answering.

**Comment by NickGravley** [comment]
In case you did not already check your ESP32 power being correct... Many PS2 to Xbox adapters have 5V where the Playstation consoles would usually have the 3.5V, and may also have 8v pin connected to 5V or not connected at all, so you'll need to use the available 5V to Vin on ESP32 rather than VCC if thats the case.

**Reply by davidxgames** [reply]
> In case you did not already check your ESP32 power being correct... Many PS2 to Xbox adapters have 5V where the Playstation consoles would usually have the 3.5V, and may also have 8v pin connected to 5V or not connected at all, so you'll need to use the available 5V to Vin on ESP32 rather than VCC if thats the case.

I already solved it, the problem was the /ATT connection, when I touch that cable with my finger, it works, I connected a 100k resistor from /ATT to 5V and that was the solution. It works correctly now.
